### DISCUSSION TITLE
Tab groups

### DISCUSSION THREAD

**Discussion by ghost** [bromite/bromite#1056] [Q&A] [answered]
Every time i want to open web page in new tab it opens in tab group. Is there any way to turn this feature off? It's a mess.
![IMG_20210410_045116_018](https://user-images.githubusercontent.com/56088966/114254524-86504c00-99e2-11eb-9842-55dbce7bacfa.jpg)
![IMG_20210410_045119_885](https://user-images.githubusercontent.com/56088966/114254525-86e8e280-99e2-11eb-9216-69e416fd8f3d.jpg)
![IMG_20210410_045123_615](https://user-images.githubusercontent.com/56088966/114254526-87817900-99e2-11eb-9940-dfe17ecde991.jpg)

**Comment by ghost** [comment, marked as answer]
![Screenshot_20210410-065939_Trebuchet](https://user-images.githubusercontent.com/79337849/114256524-559ff080-99a9-11eb-9ca9-a4bff96bdbfc.png)

**Reply by ghost** [reply]
> 
> 
> ![Screenshot_20210410-065939_Trebuchet](https://user-images.githubusercontent.com/79337849/114256524-559ff080-99a9-11eb-9ca9-a4bff96bdbfc.png)

Thank you very much!

**Reply by adatum** [reply]
> ![Screenshot_20210410-065939_Trebuchet](https://user-images.githubusercontent.com/79337849/114256524-559ff080-99a9-11eb-9ca9-a4bff96bdbfc.png)

Thank you! I like tab groups but in bromite v96 the tab groups disappeared, so by enabling those four flags I restored the functionality.
